tests/test_database_types.py

```diff
@@ -204,14 +212,22 @@ def test_types(self, source_db, target_db, source_type, target_type, type_catego
         src_table = src_conn.quote(".".join(src_table_path))
         dst_table = dst_conn.quote(".".join(dst_table_path))
 
-        src_conn.query(f"DROP TABLE IF EXISTS {src_table}", None)
-        src_conn.query(f"CREATE TABLE {src_table}(id int, col {source_type});", None)
+        if isinstance(src_conn, db.Oracle):
+            src_conn.query(f"DROP TABLE {src_table}", None)
```

Comment:
Don't we need to rescue an exception here if it doesn't support `IF EXISTS`?

Reply:
Only Oracle doesn't support if exists.

Reply:
Yes, but what about on the first run when the table doesn't exist?

Reply:
Fixed.